run_training.py

```diff
@@ -42,11 +42,15 @@ def main():
 	formatter = logging.Formatter('%(asctime)s - %(levelname)s - %(message)s')
 	console = logging.StreamHandler()
 	console.setFormatter(formatter)
+	filehandler = logging.FileHandler("debug.log", mode="w")
```

Comment:
Don't think this belongs in this PR... I agree that we could add a FileHandler (or something similar) to the standard training output, but let's do that in another PR